lib/chartmogul/customer.rb

```diff
@@ -150,6 +161,17 @@ def merge_into!(other_customer)
       true
     end
 
+    def unmerge!(data_source_uuid:, external_id:, move_to_new_customer: [])
+      options = {
+        customer_uuid: uuid,
+        data_source_uuid: data_source_uuid,
+        external_id: external_id,
+        move_to_new_customer: move_to_new_customer
+      }
+      custom!(:post, '/v1/customers/unmerges', options)
+      true
+    end
```

Comment:
[duplication] what about calling the static method?
```suggestion
    def unmerge!(data_source_uuid:, external_id:, move_to_new_customer: [])
      self.class.unmerge!(customer_uuid: uuid, data_source_uuid:, external_id:, move_to_new_customer:)
    end
```